components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/dao/SAMLSSOServiceProviderDAOImpl.java

```diff
@@ -329,8 +329,8 @@ private boolean processIsServiceProviderExists(Connection connection, String iss
 
         try (NamedPreparedStatement statement = new NamedPreparedStatement(connection,
                 SAMLSSOServiceProviderConstants.SqlQueries.GET_SAML2_SSO_CONFIG_ID_BY_ISSUER)) {
-            statement.setString(1, issuer);
-            statement.setString(2, tenantUUID);
+            statement.setString("ISSUER", issuer);
```

Comment:
Lets use constants.

Reply:
Resolved in [Update SAMLSSOServiceProviderDAOImpl and SAMLSSOServiceProviderConsta…](https://github.com/wso2/carbon-identity-framework/pull/5887/commits/827ebcb2fe58af8cbfe730557521122d0bafa0f1)